namespace.go

```diff
@@ -139,9 +139,12 @@ func NamespaceWithWebSocket() NamespaceOption {
 // The Azure Environment used can be specified using the name of the Azure Environment set in "AZURE_ENVIRONMENT" var.
 func NamespaceWithEnvironmentBinding(name string) NamespaceOption {
 	return func(ns *Namespace) error {
+		if ns.ResourceURI == "" {
+			ns.ResourceURI = serviceBusResourceURI
+		}
 		provider, err := aad.NewJWTProvider(
 			aad.JWTProviderWithEnvironmentVars(),
-			aad.JWTProviderWithResourceURI(serviceBusResourceURI),
+			aad.JWTProviderWithResourceURI(ns.ResourceURI),
```

Comment:
I'm not sure this is correct.  `JWTProviderWithEnvironmentVars()` picks up the `azure.Environment` from specified env vars which should include populating the resource URI (this is somewhat related to my above comment).

I suspect this will also require a fix in azure-amqp-common.

Reply:
I see, so should basically remove this `aad.JWTProviderWithResourceURI` call from this option, as it should all come from env.
in practice, it shouldn't hurt or change the behavior for now, since they use the same resourceURI for all env. but I think you're right